x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -126,6 +127,20 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
         },
       }),
   },
+  community_id: {
+    FieldsComponent: CommunityId,
+    docLinkPath: '/community-id-processor.html',
+    label: i18n.translate('xpack.ingestPipelines.processors.label.communityId', {
+      defaultMessage: 'Community ID',
```

Comment:
We should move this link path to the docs link service: https://github.com/elastic/kibana/blob/master/src/core/public/doc_links/doc_links_service.ts

The docs build regularly checks the docs link service for broken links.

Reply:
Oh thats great! I've created a new issue for dealing with this (see https://github.com/elastic/kibana/issues/106056) since it will involve changing the path for all the other processors and adding a few missing doc links to the service itself.